src/couch_replicator_tasks.erl

```diff
@@ -0,0 +1,52 @@
+-module(couch_replicator_tasks).
+
+-export([all/0]).
+
+-include_lib("couch/include/couch_db.hrl").
+-include("couch_replicator.hrl").
+-include("couch_replicator_api_wrap.hrl").
+
+
+all() ->
+    Jobs = ets:tab2list(couch_replicator_scheduler),
+    lists:map(fun(Job) ->
+        Rep = Job#job.rep,
+        Source = case Rep#rep.source of
+            #httpdb{} ->
+                couch_util:url_strip_password((Rep#rep.source)#httpdb.url);
+            Source0 ->
+                Source0
+        end,
+        Target = case Rep#rep.target of
+            #httpdb{} ->
+                couch_util:url_strip_password((Rep#rep.target)#httpdb.url);
+            Target0 ->
+                Target0
+        end,
+        History = lists:map(fun(Event) ->
+            EventProps  = case Event of
+                {{crashed, Reason}, _When} ->
+                    [{type, crashed}, {reason, Reason}];
+                {Type, _When} ->
+                    [{type, Type}]
+            end,
+            {_Type, {_Mega, _Sec, Micros}=When} = Event,
+            {{Y, Mon, D}, {H, Min, S}} = calendar:now_to_universal_time(When),
+            ISO8601 = iolist_to_binary(io_lib:format(
```

Comment:
We have another place that to writes timestamps replicator: https://github.com/cloudant/couchdb-couch-replicator/blob/63012-scheduler/src/couch_replicator_docs.erl#L245

There we use localtime, but I think we should be using UTC. And there we also don't include fractional seconds.  That seems reasonable, it just adds extra space or noise in the context of replication timeline (minutes, days, months will matter more here).  

In general users will monitor both the `_replicator` docs and the scheduler endpoint for the state of their replications. It would make sense to have both times be consistent (UTC probably).

Maybe we pull one timestamp(os:timestamp()) function to utils and use it from both sides? Or the change can be separate pr.